### PR TITLE
BAC 13

### PR DESCRIPTION
**Pull request by EnnEssEss**

### CODE REVIEW

**Comment by printscreen** [comment]
Looks great to me. Going to wait for @aosyborg to look it over too.

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<html ng-app="app">
+<head>
+<title>Back It</title>
+<link href="../bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
```
Comment (aosyborg) [inline src/index.html]: Can we use non-relative paths? This is something I can easily see working in development but breaking as soon as we push it out.

Comment (EnnEssEss) [inline src/index.html]: Well, currently these paths get replaced with CDN paths when making the dist build

**Comment by aosyborg** [comment]
I see karma and some unit testing setup but no actual unit tests. Should we add add at least one test just to make sure it works?

**Comment by EnnEssEss** [comment]
There's a test for the one controller that we have. See src/app/landing/landing.controller.spec.js

File: src/index.html (hunk 1 above)
Comment (aosyborg) [inline src/index.html]: Ahh missed that. Thanks!

**Pull request merged**